### PR TITLE
drop obsolete xorg drivers (bsc #1053806)

### PR DESCRIPTION
**Pull request by wfeldt**

### CODE REVIEW

File: data/root/root.file_list [hunk 1]
```diff
@@ -448,12 +448,7 @@ if arch ne 's390' && arch ne 's390x'
   ?xf86-input-vmmouse:
   xf86-input-void:
   xf86-input-wacom:
-  xf86-video-ati: nodeps
-  xf86-video-cirrus:
```
Comment (lslezak) [inline data/root/root.file_list]: Isn't Cirrus emulation used at some virtualization environment?

Comment (lslezak) [inline data/root/root.file_list]: https://linux.die.net/man/1/qemu-kvm mentions "Cirrus CLGD 5446 PCI VGA card", will KVM work correctly without that driver?

Comment (wfeldt) [inline data/root/root.file_list]: See the discussion in fate #323161. Besides, the point is moot, the packages are already gone. This is just to make installation-images build again.

Comment (lslezak) [inline data/root/root.file_list]: I see, then LGTM...

**Pull request merged**